ingestion/src/main/java/feast/ingestion/options/ImportOptions.java

```diff
@@ -119,4 +119,18 @@
   int getWindowSizeInSecForFeatureValueMetric();
 
   void setWindowSizeInSecForFeatureValueMetric(int seconds);
+
+  @Description(
+      "Set the Redis key TTL based on the max age of the FeatureSet that data is being written for.")
+  @Default.Boolean(false)
+  boolean getEnableRedisTTL();
```

Comment:
this changes can be reverted, since configuration moved to `RedisClusterConfig`

Reply:
@pyalex Done.